operators/trello_to_s3_operator.py

```diff
@@ -0,0 +1,145 @@
+from airflow.hooks.S3_hook import S3Hook
+from airflow.models import BaseOperator
+
+from trello_plugin.hooks.trello_hook import TrelloHook
+
+from tempfile import NamedTemporaryFile
+import json
+
+class TrelloToS3Operator(BaseOperator):
+    """
+    Trello to S3 Operator
+    :param trello_conn_id:          The Airflow id used to store the Trello
+                                    credentials.
+    :type trello_conn_id:           string
+    :param endpoint:                The endpoint to retrive data from.
+                                    Implemented for: 
+                                        - boards
+                                        - cards
+                                        - actions 
+                                        - lists
+                                        - checklists
+                                        - members
+    :type endpoint:                 string
+    :param s3_conn_id:              The Airflow connection id used to store
+                                    the S3 credentials.
+    :type s3_conn_id:               string
+    :param s3_bucket:               The S3 bucket to be used to store
+                                    the Marketo data.
+    :type s3_bucket:                string
+    :param s3_key:                  The S3 key to be used to store
+                                    the Marketo data.
+    :type s3_bucket:                string
+    :param fields:                  List of fields to retrive for each object
+                                    from that endpoint
+    :type fields:                   list
+    :param since:                   The starting date parameter.
+    :type since: 
+    :param: before:                 The ending date parameter.
+    :type before:                   
+    """
+
+    def __init__(self,
+                 trello_conn_id,
+                 endpoint,
+                 s3_conn_id,
+                 s3_bucket,
+                 s3_key,
+                 fields=None,
+                 since=None,
+                 before=None,
+                 *args,
+                 **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.trello_conn_id = trello_conn_id
+        self.endpoint = endpoint
+        
+        self.since = since
+        self.before = before
+        self.fields = fields
+        
+        self.s3_conn_id = s3_conn_id
+        self.s3_bucket = s3_bucket
+        self.s3_key = s3_key
+        
+        self.hook = TrelloHook(http_conn_id=self.trello_conn_id)
+
+        if self.endpoint not in (
+            'boards',
+            'actions',
+            'cards',
+            'checklists',
+            'lists',
+            'members'):
+            raise Exception('Specified endpoint not currently supported.')
```

Comment:
Personally I would prefer to define my own class that inherits from `Exception` here instead of doing this. 

```Python
class EndpointNotSupported(Exception):

    def __init__(self) -> None:
        super().__init__("Specified endpoint not currently supported.")
```
and then use it here like this. 
```Python
raise EndpointNotSupported()
```